Corregido el error ##11265 [es] (#11353)
mdn/translated-content: files/es/mdn/community/contributing/translated_content/index.md

---
title: Localización de MDN Web Docs
slug: MDN/Community/Contributing/Translated_content
page-type: mdn-community-guide
l10n:
  sourceCommit: 4c6b993ad0d43bc679688a3488e13a4ec0b2c3ec
---

{{MDNSidebar}}

> Desde el 14 de diciembre de 2020, MDN corre en la nueva [pataforma Yari](https://github.com/mdn/yari), basada en GitHub. Esto tiene muchas ventajas para MDN, pero hemos tenido que hacer cambios radicales en la forma en que manejamos la localización. Esto se debe a que hemos terminado con una gran cantidad de contenido no mantenido y desactualizado en nuestras localizaciones diferentes a en-Us, y queremos gestionarlo mejor en el futuro.

Reviewer: ```suggestion
Desde el 14 de diciembre de 2020, MDN corre en la nueva [plataforma Yari](https://github.com/mdn/yari), basada en GitHub. Esto tiene muchas ventajas para MDN, pero hemos tenido que hacer cambios radicales en la forma en que manejamos la localización. Esto se debe a que hemos terminado con una gran cantidad de contenido no mantenido y desactualizado en nuestras localizaciones diferentes a en-US, y queremos gestionarlo mejor en el futuro.
```

> Hemos congelado todo el contenido localizado (lo que significa que no aceptaremos ninguna edición del mismo, quedando éste como contenido de sólo lectura), EXCEPTO para las localizaciones que se muestran debajo. Estas localizaciones tienen equipos dedicados que se responsabilizan de su mantenimiento.

Reviewer 2: Ya que es un modismo

```suggestion
Hemos _congelado_ todo el contenido localizado (lo que significa que no aceptaremos ninguna edición del mismo, quedando éste como contenido de sólo lectura), EXCEPTO para las localizaciones que se muestran debajo. Estas localizaciones tienen equipos dedicados que se responsabilizan de su mantenimiento.
```

- Colaboradores actuales: [mfuji09](https://github.com/mfuji09), [hmatrjp](https://github.com/hmatrjp), [potappo](https://github.com/potappo), [dynamis](https://github.com/dynamis), [kenji-yamasaki](https://github.com/kenji-yamasaki)

### Coreano (ko)

- Discusiones: [Kakao Talk (#MDN Korea)](https://open.kakao.com/o/gdfG288c)
- Current contributors: [hochan222](https://github.com/hochan222), [yechoi42](https://github.com/yechoi42), [cos18](https://github.com/cos18), [GwangYeol-Im](https://github.com/GwangYeol-Im), [pje1740](https://github.com/pje1740), [yujo11](https://github.com/yujo11)

### Ruso (ru)

- Discusiones: [Matrix (canal #mdn-l10n-ru)](https://chat.mozilla.org/#/room/#mdn-l10n-ru:mozilla.org)
- Colaboradores actuales: [armanpwnz](https://github.com/armanpwnz), [captainspring](https://github.com/captainspring), [myshov](https://github.com/myshov), [Saionaro](https://github.com/Saionaro), [sashasushko](https://github.com/sashasushko), [lex111](https://github.com/lex111)

### Español (es)

> - Discusiones: [Matrix (canal #mdn-l10n-es)](https://chat.mozilla.org/#/room/#mdn-l10n-es:mozilla.org)

Reviewer 2: ```suggestion
- Discusiones: [Matrix (canal #mdn-l10n-es)](https://chat.mozilla.org/#/room/#mdn-l10n-es:mozilla.org), [Telegram (canal MDN l10n ES)](https://t.me/+Dr6qKQCAepw4MjFj), [Discord (MDN Web Docs Community ,canal #spanish)](https://discord.gg/MkEg2ATW)
```

> - Colaboradores actuales: [JuanVqz](https://github.com/JuanVqz), [davbrito](https://github.com/davbrito), [Graywolf9](https://github.com/Graywolf9), [Vallejoanderson](https://github.com/Vallejoanderson), [marcelozarate](https://github.com/marcelozarate), [Jalkhov](https://github.com/Jalkhov)

Reviewer: Para comenzar, si @Graywolf9 esta de acuerdo creo que podemos proceder con esta sugerencia de lo contrario está bien como lo tradujiste 👍

```suggestion
- Colaboradores actuales: [JuanVqz](https://github.com/JuanVqz), [Graywolf9](https://github.com/Graywolf9), [marcelozarate](https://github.com/marcelozarate), [Jalkhov](https://github.com/Jalkhov)
```

Reviewer 2: Creo que este puede esperar para hacer el proceso completo y que esto indique lo mismo que los permisos de github

> - Colaboradores actuales: [JuanVqz](https://github.com/JuanVqz), [davbrito](https://github.com/davbrito), [Graywolf9](https://github.com/Graywolf9), [Vallejoanderson](https://github.com/Vallejoanderson), [marcelozarate](https://github.com/marcelozarate), [Jalkhov](https://github.com/Jalkhov)

Reviewer 2: Just reorder

```suggestion
- Colaboradores actuales: [Graywolf9](https://github.com/Graywolf9), [JuanVqz](https://github.com/JuanVqz), [Jalkhov](https://github.com/Jalkhov), [marcelozarate](https://github.com/marcelozarate), [davbrito](https://github.com/davbrito), [Vallejoanderson](https://github.com/Vallejoanderson).
```

> > **Nota:** Si quiere hablar sobre la posibilidad de descongelar una localización actualmente congelada, las [directrices sobre lo que se requiere se pueden encontrar aquí](https://github.com/mdn/translated-content/blob/main/PEERS_GUIDELINES.md#activating-a-locale)

Reviewer 2: ```suggestion
> **Nota:** Si quiere hablar sobre la posibilidad de _descongelar_ una localización, las [directrices sobre lo que se requiere se pueden encontrar aquí](https://github.com/mdn/translated-content/blob/main/PEERS_GUIDELINES.md#activating-a-locale)
```

## Véase también

- [Actualización de la localización de MDN, febrero de 2021](https://hacks.mozilla.org/2021/02/mdn-localization-update-february-2021/) - el último estado de la localización en MDN

> - [Una actualización de la estrategia de localización de MDN Web Docs](https://hacks.mozilla.org/2020/12/an-update-on-mdn-web-docs-localization-strategy/) - estrategia actualizada basada en las reacciones de la comunidad

Reviewer 2: ```suggestion
- [Una actualización de la estrategia de localización de MDN Web Docs](https://hacks.mozilla.org/2020/12/an-update-on-mdn-web-docs-localization-strategy/) - estrategia actualizada basada en los comentarios de la comunidad
```
